architecture/core_concepts/projects_and_users.adoc

```diff
@@ -134,3 +134,12 @@ endif::[]
 Developers and administrators can xref:../../dev_guide/projects.adoc#dev-guide-projects[interact
 with projects] using xref:../../cli_reference/index.adoc#cli-reference-index[the CLI] or the
 xref:../infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console].
+
+ifdef::openshift-online[]
+[[projects-hibenation]]
+=== Project Hibernation
+In {product-title} Starter, projects hibernate after 30 minutes of inactivity.
```

Comment:
@ahardin-rh @sallyom 
1. Whatever the hours in the [design](https://docs.google.com/document/d/1-oJsAqMfcZdC8I_7P3JnGOfRMDcSKm3OnSKmhY5RM1o/edit#heading=h.pr8fz6j7jwnm) or the [introduction](https://github.com/openshift/online/blob/master/force-sleep/Readme.md),  the hibernate is 8 hours and the period is 24 hours,  but at here, they are 18 of 72, are our design changed to this?
2. About the last sentence,  from the testing on devenv, the project will be given an extra quota force-sleep which only hard code the pod limit to zero, not all the resources(dc,rc,pvc can be successfully created).